documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -84,7 +84,7 @@ To enable the `UseKRaft` feature gate, specify `+UseKRaft` in the `STRIMZI_FEATU
 
 The `StableConnectIdentities` feature gate has a default state of _disabled_.
 
-The `StableConnectIdentities` feature gate uses `StrimziPodSet` resources to manage Kafka Connect and Kafka MirrorMaker 2.0  pods instead of using Kubernetes Deployment resources.
+The `StableConnectIdentities` feature gate uses `StrimziPodSet` resources to manage Kafka Connect and Kafka MirrorMaker 2  pods instead of using Kubernetes Deployment resources.
```

Comment:
```suggestion
The `StableConnectIdentities` feature gate uses `StrimziPodSet` resources to manage Kafka Connect and Kafka MirrorMaker 2 pods instead of using Kubernetes Deployment resources.
```